fv3core/utils/corners.py

```diff
@@ -352,6 +409,124 @@ def copy_corners_y_stencil_defn(q_in: FloatField, q_out: FloatField):
             q_out = q_in[-3, -2, 0]
 
 
+def copy_corners_xy_stencil_defn(
+    q_in: FloatField, q_out_x: FloatField, q_out_y: FloatField
+):
+    from __externals__ import i_end, i_start, j_end, j_start
+
+    with computation(PARALLEL), interval(...):
+        q_out_x = q_in
+        q_out_y = q_in
+        with horizontal(
+            region[i_start - 3, j_start - 3], region[i_end + 3, j_start - 3]
+        ):
+            q_out_x = q_in[0, 5, 0]
```

Comment:
I thought we had found this pattern to not be ok/not work. But I see it was changed to this in the single x/y  variables so I'll follow up elsewhere